apps/store/modules/page-decorators.js

```diff
@@ -161,13 +161,13 @@ var pageDecorators = {};
                 childValues = categorizationField.values[0].value;
             }
 
-            if(index < ctx.rxtManager.collapseInCount(ctx.assetType)){
+            if (index < ctx.rxtManager.collapseInCount(ctx.assetType)){
                 updatedCategorizationField.isCollapseIn = true;
             } else {
                 updatedCategorizationField.isCollapseIn = false;
             }
 
-            for(var childIndex in childValues){
+            for(var childIndex=0; childIndex < childValues.length; childIndex++){
```

Comment:
Thanks for pointing it out. Fixed with the following PR - https://github.com/wso2/carbon-store/pull/833